Music-Player-App/main.py

```diff
@@ -0,0 +1,263 @@
+
+
+def about_us():
+    tkinter.messagebox.showinfo(
+        'About Rockerz', 'This Is A Music Player Devloped With Python Tkinter And Pygame By Robin Singh')
+
+
+def browse_files():
+    global music_file
+    music_file = filedialog.askopenfilename()
+    add_to_listbox(music_file)
+
+
+music_list = []
+
+
+def add_to_listbox(music_file):
+    file_name = os.path.basename(music_file)
+    index = 0
+    play_list.insert(index, file_name)
+    music_list.insert(index, music_file)
+    play_list.pack()
+    index += 1
+
+
+def delete_btn():
+    song = play_list.curselection()
+    song = int(song[0])
+    play_list.delete(song)
+    music_list.pop(song)
+
+
+def mute_music():
+    global muted
+    if muted:
+        mixer.music.set_volume(.7)
+        vol_button1.configure(image=pic5)
+        scale1.set(70)
+        muted = FALSE
+
+    else:
+        mixer.music.set_volume(0)
+        vol_button1.configure(image=pic4)
+        scale1.set(0)
+        muted = TRUE
+
+
+def close_window_fully1():
+    Stop_music()
+    exit()
+
+
+muted = FALSE
+
+
+main_window = tk.ThemedTk()
+main_window.get_themes()
+# themes : 'arc','radiance','breeze','ubuntu' etc
+main_window.set_theme("breeze")
+# creating toolbar
+tool_bar = Menu(main_window)
+main_window.config(menu=tool_bar)
+
+status_bar = ttk.Label(main_window, text="Welcome To Rockerzz",
+                       relief=SUNKEN, anchor=W, font='verdana 10 italic')
+status_bar.pack(side=BOTTOM, fill=X)
+
+# creating sub menus
+sub_menu = Menu(tool_bar, tearoff=0)  # to remove dashed line from menu
+tool_bar.add_cascade(label='File', menu=sub_menu)
+sub_menu.add_command(label="Open", command=browse_files)
+sub_menu.add_command(label="Exit", command=close_window_fully1)
+
+sub_menu = Menu(tool_bar, tearoff=0)  # to remove dashed line from menu
+tool_bar.add_cascade(label='Help', menu=sub_menu)
+sub_menu.add_command(label="About Us ", command=about_us)
+
+
+mixer.init()
+
+# main_window.geometry("600x300")
+main_window.title("Rockerz")
+main_window.iconbitmap(r"assests/rockerz.ico")
+
+left_frame = Frame(main_window)
+left_frame.pack(side=RIGHT, padx=30, pady=20)
+play_list = Listbox(left_frame)
+play_list.pack()
+
+
+add_btn = ttk.Button(left_frame, text='ADD', command=browse_files)
+add_btn.pack(side=LEFT, padx=3)
+
+del_btn = ttk.Button(left_frame, text='DELETE', command=delete_btn)
+del_btn.pack(side=LEFT)
+
+right_frame = Frame(main_window)
+right_frame.pack(pady=20)
+
+r_top_frame = Frame(right_frame)
+r_top_frame.pack()
+
+Main_text = ttk.Label(
+    r_top_frame, text="Devloped By Robin Singh", font='arial 10 italic')
+Main_text.pack()
+
+Main_lenth = ttk.Label(r_top_frame, text="Length : --:--", relief=GROOVE)
+Main_lenth.pack(pady=5)
+
+current_lenth = ttk.Label(
+    r_top_frame, text="Current Duration : --:--", relief=GROOVE)
+current_lenth.pack()
+
+playlist_box = Listbox(main_window)
+canvas = Frame(right_frame)
+canvas.pack(pady=5)
+
+pic = PhotoImage(file="assests/images/play.png")
```

Comment:
The images are still not loading and getting the error. 

You need to give path as "./Music-Player-App/assests..." everywhere you want to load images.